### PR TITLE
 ISPN-14955 LTRIM

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14955

### CODE REVIEW

File: server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java [hunk 1]
```diff
@@ -355,4 +355,25 @@ public void testLREM() {
...
+      assertThat(redis.lrange("leads", 0, -1)).containsExactly("jose");
+      assertThat(redis.ltrim("leads", 1, -1)).isEqualTo("OK");
+      assertThat(redis.exists("leads")).isEqualTo(0);
+
+      // Set a String Command
+      redis.set("another", "tristan");
+
+      // LLEN on an existing key that contains a String, not a Collection!
+      assertThatThrownBy(() -> {
+         redis.llen("another");
```
Comment (jabolina) [inline server/resp/src/test/java/org/infinispan/server/resp/RespListCommandsTest.java:375]: I guess `ltrim` here.

**Review by jabolina — COMMENTED** [review]
Just one small thing.

**Comment by karesti** [comment]
@jabolina changes reported

**Comment by jabolina** [comment]
Failures do not seem related.

**Pull request merged**